rust/kernel/lib.rs

```diff
@@ -16,7 +16,9 @@
 #![feature(coerce_unsized)]
 #![feature(core_ffi_c)]
 #![feature(dispatch_from_dyn)]
+#![feature(explicit_generic_args_with_impl_trait)]
 #![feature(generic_associated_types)]
+#![feature(new_uninit)]
```

Comment:
should these feature enables be in their own commits?

Reply:
Don't worry about it -- Lina has a patch enabling it too, so I can pick that one first: https://lore.kernel.org/rust-for-linux/20230224-rust-new_uninit-v1-1-c951443d9e26@asahilina.net/